linkis-dist/package/admin/linkis-ec-clear.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#The cleaning date must be passed in. And ecm will print the actual effective date log.
+end_date=`date -d '-90 days'`
+end_date_t=`date -d '-90 days' +%s`
+echo "Clean up data before this date:$end_date"
+#Directory read ecm engine cache variables
+prefix=/data/bdp/linkis/
```

Comment:
need to update install.sh replace prefix

Reply:
done